src/doc/reference/src/visibility-and-privacy.md

```diff
@@ -40,7 +40,10 @@ With the notion of an item being either public or private, Rust allows item
 accesses in two cases:
 
 1. If an item is public, then it can be used externally through any of its
-   public ancestors.
+   accessible ancestors (either by anyone that has an access to an ancestor
+   in the case of a chain of public ancestors,
+   or by the siblings of a private ancestor, in the case of a chain of public
+   ancestors terminating into a private ancestor).
```

Comment:
I think the sentence you added is better worded as something like "you can directly name the item from some module `m` if you can name all the item's parent modules from `m`" plus "you can potentially name the `pub` item indirectly through reexports from any other modules", but I still may be more useful to describe the intent of the rules instead.
(Removing the incorrect paragraph 1 is desirable in any case.)